src/components/hint/__tests__/index.spec.tsx

```diff
@@ -1,69 +1,128 @@
+      let contentColor = await driver.getContentStyle().backgroundColor;
+      expect(contentColor).toBe(expectedColor);
 
-    const driver = new HintDriver({component, testID: 'Hint'});
-    expect(await driver.getElement()).toBeTruthy();
+      const whiteHintRenderTree = render(<HintTestComponent showHint color={Colors.white}/>);
+      const whiteHintDriver = HintDriver({renderTree: whiteHintRenderTree, testID: TEST_ID});
 
-    const hintContent = await driver.getHintContent();
-    let color = findStyle('backgroundColor', hintContent);
-
-    expect(color).toBe(expectedColor);
-    expect(hintContent).toBeTruthy();
+      contentColor = await whiteHintDriver.getContentStyle().backgroundColor;
+      expect(contentColor).toBe(Colors.white);
+    });
+  });
 
-    const WhiteHint = <HintTestComponent showHint color={Colors.white}/>;
-    const WhiteHintDriver = new HintDriver({component: WhiteHint, testID: 'Hint'});
+  describe('Test Hint icon', () => {
+    it('Hint should include icon', async () => {
+      const renderTree = render(<HintTestComponent showHint useIcon/>);
+      const driver = HintDriver({renderTree, testID: TEST_ID});
+      expect(driver.getIcon().exists()).toBeTruthy();
+    });
 
-    const WhiteHintContent = await WhiteHintDriver.getHintContent();
-    color = findStyle('backgroundColor', WhiteHintContent);
-    expect(color).toBe(Colors.white);
+    it('Hint shouldn\'t include icon', async () => {
+      const renderTree = render(<HintTestComponent showHint/>);
+      const driver = HintDriver({renderTree, testID: TEST_ID});
+      expect(driver.getIcon().exists()).toBeFalsy();
+    });
   });
 
-  it('Test Hint modal is not visible when showHint is false', async () => {
-    const component = <HintTestComponent showHint={false}/>;
-    const driver = new HintDriver({component, testID: 'Hint'});
-    expect(await driver.getHintModal()).toBeFalsy();
+  describe('Test Hint modal visibility:', () => {
+    it('Test Hint modal is not visible when showHint is false', async () => {
+      const renderTree = render(<HintTestComponent showHint={false}/>);
+      const driver = HintDriver({renderTree, testID: TEST_ID});
+      expect(await driver.getModal().exists()).toBeFalsy();
+    });
+
+    it('Test Hint modal is visible when showHint is true', async () => {
+      const renderTree = render(<HintTestComponent showHint onBackgroundPress={() => {}}/>);
+      const driver = HintDriver({renderTree, testID: TEST_ID});
+      expect(await driver.exists()).toBeTruthy();
+      expect(await driver.getModal().exists()).toBeTruthy();
+    });
   });
 
-  it('Test Hint modal is visible when showHint is true', async () => {
-    const component = <HintTestComponent showHint/>;
-    const driver = new HintDriver({component, testID: 'Hint'});
-    expect(await driver.getElement()).toBeTruthy();
-    expect(await driver.getHintModal()).toBeTruthy();
+  describe('Test Hint onPress & onBackgroundPress', () => {
```

Comment:
These do not have to be together, if you have only a single test for `onBackgroundPress` it can still be on it's own (either as an `it` or as a `describe`)